Project-Uni/RootShare
# Create Admin Hub

**Note!!** - Branched off pending-requests PR
**Updates**
- Creates a Hub for Admins to easily navigate through admin pages
- Moves all admin frontend routes into new routing component for improved maintanability

**Testing**
- Go to /admin. If you are not an admin, it should show an invalid message. If you are an admin it should show the links.
- Click through each link and see that it navigates properly
- Resize page to see if any sizing issues
